documentation/modules/ROOT/pages/configuration/event-flattening.adoc

```diff
@@ -55,114 +101,247 @@ The downside of using the complex format is that other connectors or other parts
 }
 ----
 
-Debezium provides {link-kafka-docs}/#connect_transforms[a single message transformation] that crosses the bridge between the complex and simple formats, the https://github.com/debezium/debezium/blob/master/debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordState.java[ExtractNewRecordState] SMT.
+To provide the needed Kafka record format for consumers, configure the `ExtractNewRecordState` SMT.
+
+ifdef::community[]
+== Behavior
+endif::community[]
+
+ifdef::product[]
+// Type: concept
+[id="behavior-of-debezium-extractnewrecordstate-transformation"]
+== Behavior of Debezium `ExtractNewRecordState` transformation
+endif::product[]
```

Comment:
I have seen this in multiple spots i the changed files in this PR and I'm not sure this differentiation is needed.  In the MySQL documentation as an example, a number of headings that were much shorter in the original writings are now much more verbose and resemble what you have scoped as product here.  I'm inclined to think we can omit the separation between community/product and merely use the more verbose section headings.

Any preference @gunnarmorling?

Reply:
Take a look at this page: https://debezium.io/documentation/reference/1.0/connectors/mysql.html
The entries on the right are impossible to scan quickly. A user has to read them. I'm trying to provide a quicker way for the user to find the information they want. When the Event Flattening page appears,  in the right toc panel, do you want the user to see this: 

```
Change event structure
Behavior
Configuration
Example
Options
```
Or this: 
```
Description of Debezium change event structure
Behavior of Debezium ExtractNewRecordState transformation
Configuration of ExtractNewRecordState transformation
Example of adding metadata to the Kafka record or its header
Options for configuring ExtractNewRecordState transformation
```
I think the first option provides a better user experience. Upstream, the user does not need the context that is required to be provided in the downstream headings. But I will do whichever you prefer.

Reply:
So I'm not 100% up-to-date on the latest state of this discussion, but my preference would definitely be on the shorter, more concise captions in the upstream ToC on the right-hand side of docs. If that gets enabled by this PR, I'm +1 on that.